app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    API_KEY = os.environ["API_KEY"]
+    URL = "https://api.weatherapi.com/v1/current.json?"
+    CITY = "Paris"
```

Comment:
don't you think that constant variables are better to have in global scope?

Reply:
Fixed